cdci_data_analysis/analysis/queries.py

```diff
@@ -529,7 +524,9 @@ def get_query_products(self,instrument,job,run_asynch,query_type='Real',logger=N
             logger.error("passing request issue: %s", e)
             raise
 
-        except Exception as e: # noo!!!
+        except Exception as e: # TODO: could we avoid these? they make error tracking hard
+            logger.exception("failed to get query products")
```

Comment:
This is indeed a bit too generic is such an important stage of the whole processing. Perhaps the definition of a dedicated exception could help in tracking down the error

Reply:
indeed. This is also related to https://github.com/oda-hub/dispatcher-app/issues/25 . But I do not want to address the entire challenge in this PR.